src/sail_sv_backend/jib_sv.ml

```diff
@@ -402,17 +406,19 @@ let collect_spec_info ctx cdefs =
       )
       IdSet.empty cdefs
   in
-  let global_lets, global_let_numbers =
+  let global_lets, global_let_numbers, globals =
```

Comment:
If we need the `ctyp` here, it would be cleaner to just turn `global_lets` into a `NameMap` and refactor that way rather than duplicating everything.